client/lib/plans/index.js

```diff
@@ -219,6 +220,10 @@ export function planLevelsMatch( planSlugA, planSlugB ) {
 	return planA && planB && planA.type === planB.type && planA.group === planB.group;
 }
 
+export function isEcommercePlan( planSlug ) {
```

Comment:
What is the difference between `isEcommercePlan` and `isWpComEcommercePlan`? Won't all ecommerce plans in the context of Claypso always be `isWpComEcommercePlan` too?

Reply:
Yes, you're right, that one is redundant. I'm going to remove it.

Reply:
On the other hand, now that I am trying to remove it, I think it's not as redundant as I initially thought. In this case `isEcommercePlan` checks for eCommerce type, while `isWpComEcommercePlan` checks also for specific WordPress.com plans. I think it might make sense to keep that check too just to mirror other selectors and match code expectations. What do you think?

Reply:
Yeah after looking at the patterns in this file, this seems like a common approach. I have never looked at the plan code in calypso before so that is why i found it odd at first. So LGTM as it stands.